src/features/404/index.tsx

```diff
@@ -1,11 +1,27 @@
 import { Link } from 'react-router-dom';
 import { PLANET_OBSERVER_PATH } from 'src/features/Router/path';
 
-const Page404 = () => (
-  <div>
-    <h1>404</h1>
-    <Link to={PLANET_OBSERVER_PATH}>Go to home page</Link>
-  </div>
-);
-
-export default Page404;
+import { PageHeader } from 'src/components/PageHeader/PageHeader';
+import { BurgerIcon } from 'src/components/_common/icons/BurgerIcon'
+
+const Page404 = () => {
+
+  const burgerIconProps = {
+    color: '',
+    fontSize: 20,
+  };
+
+  return (
+    <>
+    <PageHeader>
+      <BurgerIcon {...burgerIconProps}/>
```

Comment:
Why do you use spred for 2 simple props? And ony of the is emty? You also forget URL for link
```
<Link to="..."><BurgerIcon fontSize="..." /></Link>
```

Reply:
Changed